static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-immersive.scss

```diff
@@ -217,5 +208,19 @@
                 @include immersiveSublinks();
             }
         }
+
+        @include mq($from: tablet) {
```

Comment:
Shouldn't this also be wrapped inside a card type class selector? I was under the impression that immersive cards worked in mobile too, except when in list-type cards, but I may be wrong 😅

Reply:
You're sharp-eyed even at this ungodly hour! Thanks man, I'm on it